src/osd/OSD.cc

```diff
@@ -2153,6 +2153,8 @@ void OSD::create_logger()
 
   osd_plb.add_u64_counter(l_osd_copyfrom, "copyfrom", "Rados \"copy-from\" operations");
 
+  osd_plb.add_u64_counter(l_osd_tier_total, "tier_total", "Tier total ops");
+  osd_plb.add_u64_counter(l_osd_tier_hit, "tier_hit", "Tier hits");
   osd_plb.add_u64_counter(l_osd_tier_promote, "tier_promote", "Tier promotions");
```

Comment:
Originally, I add it. But find it duplicated with l_osd_tier_promote. Please refer to xinze's comment above